types/p5/src/color/setting.d.ts

```diff
@@ -4,6 +4,64 @@ import * as p5 from '../../index';
 
 declare module '../../index' {
     interface p5InstanceExtensions {
+        /**
+         *   Start defining a shape that will mask subsequent
+         *   things drawn to the canvas. Only opaque regions of
+         *   the mask shape will allow content to be drawn. Any
+         *   shapes drawn between this and endClip() will
+         *   contribute to the mask shape. The mask will apply
+         *   to anything drawn after this call. To draw without
+         *   a mask, contain the code to apply the mask and to
+         *   draw the masked content between push() and pop().
+         *
+         *   Alternatively, rather than drawing the mask
+         *   between this and endClip(), draw the mask in a
+         *   callback function passed to clip().
+         *
+         *   Options can include:
+         *
+         *   - invert: A boolean specifying whether or not to
+         *   mask the areas not filled by the mask shape.
+         *   Defaults to false.
+         *   @param [options] An object containing clip
+         *   settings.
+         */
+        beginClip(options?: object): void;
```

Comment:
What are the allowed `options`?  I know that this is an auto-generated types file and that change needs to be made in the root p5.js package but I would love to see a `@typedef` of `ClipOptions`!